bin/dev_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+echo "Checking java 1.8.x"
```

Comment:
What we have here is specific, yes.  It follows the README, last time I checked.  Feel free to modify it.  Note that the grep is not fussy about the minor release number and I made no attempts to double check the README against any of the jetty/Solr/Fedora packages.